client/templates/app/nav/nav.html

```diff
@@ -10,11 +10,10 @@
           {{> notificationsButton}}
           {{> userDropdown}}
         {{else}}
-          <li><a href="{{pathFor 'login'}}">Sign in</a></li>
-          <li><a href="{{pathFor 'signup'}}">Register</a></li>
+          <li><a href="/signup">Login / Register</a></li>
```

Comment:
OK.  I just did it like that because the way this package is built is it uses just the one "atForm" and then maintains which state it's in.  I don't know if separating "login" and "register" is worth trying to jump through hoops with how the package is designed really imho.